docker/docker.go

```diff
@@ -98,6 +101,9 @@ func DoRunContainer(ctx context.Context, dClient *client.Client, imageCfg *conta
 }
 
 func DoRollingUpdateContainer(ctx context.Context, dClient *client.Client, imageCfg *container.Config, hostCfg *container.HostConfig, containerName, hostname, plane string, prsMap map[string]v3.PrivateRegistry) error {
+	if dClient == nil {
+		return fmt.Errorf("[%s] DoRollingUpdateContainer client is nil for container [%s] on host [%s]", plane, containerName, hostname)
```

Comment:
Can you make the error more descriptive, like `Failed to roll update the container: docker client is nil`